dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -984,6 +1025,13 @@
           "items": {
             "$ref": "#/definitions/HealthcareEntityLink"
           }
+        },
+        "EntityComponentInformation": {
+          "description": "(Optional) Entity component information listing fired components of the extracted entity.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EntityComponentInformation"
+          }
```

Comment:
I think the name of the model being used for each item should be different from the parent one.

Reply:
Ah the capitalization threw me off and I thought it was a standalone definition, never mind this comment.